conda_forge_tick/make_graph.py

```diff
@@ -352,6 +352,25 @@ def update_nodes_with_bot_rerun(gx):
                     )
 
 
+def update_nodes_with_new_versions(gx):
+    """Updates every node with it's new version (when available)"""
+    # check if the versions folder is available
+    if os.path.isdir("./versions"):
+        # get all the available node.json files
+        pass
+    else:
+        return
+    list_files = os.listdir("./versions/")
+    for file in list_files:
+        node = str(file).replace(".json", "")
```

Comment:
why are you removing `.json`?

Reply:
To compute the node name in the graph I think

Reply:
To get the node name to update the attrs information regarding that node